core/src/main/java/org/sonarsource/sonarlint/core/embedded/server/ShowHotspotRequestHandler.java

```diff
@@ -55,21 +53,20 @@
 
 @Named
 @Singleton
-public class ShowHotspotRequestHandler implements HttpRequestHandler {
+public class ShowHotspotRequestHandler extends ShowHotspotOrIssueRequestHandler implements HttpRequestHandler {
```

Comment:
I think it's ok here, but just for sharing, I think it's a common practice to favor [composition over inheritance](https://en.wikipedia.org/wiki/Composition_over_inheritance). This could be achieved here by having something like a `BindingAssistantService` that would be used by both handlers